src/app/shared/menu/export/export-dropdown.component.ts

```diff
@@ -5,10 +5,12 @@ import { Observable } from 'rxjs/Observable';
 import { map } from 'rxjs/operators/map';
 import { GoogleAnalyticsEventAction } from '../../../ngrx/actions';
 import { selectExportData2DArray, joinCsvArray } from '../../../ngrx/reducers/selectors';
+import { ExportGoogleSheetsService } from './export-google-sheets.service';
 
 @Component({
   selector: 'metrics-export-dropdown',
   template: `
+    <prx-spinner *ngIf="googleSheetsBusy$ | async" overlay="true"></prx-spinner>
```

Comment:
I made the ExportGoogleSheetsService into an observable data service just so this dang spinner could live out here in the DOM. I supposed I could have used NgRx here and made the sheets export an effect, but putting this in application state seemed unnecessary and misplaced. It's true, "you might not need redux."